jss.spec.in

```diff
@@ -68,18 +65,12 @@ This package contains the API documentation for JSS.
 ################################################################################
 %prep
 
-%autosetup -n %{name}-%{version} -p 1 -S git
-
-# Prior to version 4.4.5, the source code were stored under "jss-<version>/jss"
-# path in the source tarball. Starting from version 4.4.5, the files will be
-# stored under "jss-<version>" path. However, since the spec file is still using
-# the old path, the unpacked source code has to be moved to the old path with
-# the following commands.
-
-cd ..
-mv %{name}-%{version} jss
-mkdir %{name}-%{version}
-mv jss %{name}-%{version}
+#%autosetup -n %{name}-%{version} -p 1 -S git
```

Comment:
Is it not possible to use autosetup with 4.4-style tarball?

Reply:
This is a quirk of the `v4.4.x` style tarball coupled with a hack to enable `git format-patch` again iirc (or, `git diff` as I tend to do) I think. What we do is `pushd` into the `jss/` subdir, apply patches, then `popd` back out and continue on. This lets us generate "regular" patches from inside the git repo and not worry about adding the extra `jss/` subpath.